File: Unter/src/main/java/com/atai/unter/module/enterprise/model/Address.java
package com.atai.unter.module.enterprise.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "address_tab")
public class Address {

	@Id
	@Column(name = "address_id")
	private int addressId;
	
	private String street;
	
	private String city;
	
	private String country;

	public int getAddressId() {
		return addressId;
	}

	public void setAddressId(int addressId) {
		this.addressId = addressId;
	}

	public String getStreet() {
		return street;
	}

	public void setStreet(String street) {
		this.street = street;
	}

	public String getCity() {
		return city;
	}

	public void setCity(String city) {
		this.city = city;
	}

	public String getCountry() {
		return country;
	}

	public void setCountry(String country) {
		this.country = country;
	}
	
	
}

File: Unter/src/main/webapp/static/js/entity/Address.js
'use strict';
var entityModule = angular.module('entityModule',[]);
entityModule.factory('EntityService', ['$http', '$q', function($http, $q){

    var entityRec = {
        name   :'Address',
        record :{addressId:'',street:'',city:'',country:'',objid:null},
        emptyRecord :emptyRecord
    };
    return entityRec;
    
    function emptyRecord() {
        return {addressId:'',street:'',city:'',country:'',objid:null};
    }	
}]);